handlers/BusRiderNumberFinderIntentHandler.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
+const { BUS_ID_TO_NAME, SKILL_NAME } = require('../utilities/constants/constants');
+
+const BusRiderNumberFinderIntentHandler = {
+  canHandle(handlerInput) {
+    return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+        && handlerInput.requestEnvelope.request.intent.name === 'BusRiderNumberFinderIntent';
+  },
+
+  async handle(handlerInput) {
+    // sets slot values from AWS Alexa to stopId
+    const slotId = handlerInput.requestEnvelope.request.intent.slots.BUS_ROUTE.resolutions.resolutionsPerAuthority[0].values[0].value.id
+    // const busInfo = BUS_ID_TO_NAME[slotId];
+    // const routeData = await axios.get(`https://realtime.catabus.com/InfoPoint/rest/Vehicles/Get/${busInfo.slotId}`).then(response => response.data);
+    const routeData = await axios.get(`https://realtime.catabus.com/InfoPoint/rest/RouteDetails/Get/1`).then(response => response.data);
+    const numBuses = routeData.Vehicles.length;
+    const numberOfRiders = routeData.Vehicles.map(vehicle => vehicle.OnBoard);
+    console.log(numberOfRiders.toString());
```

Comment:
Shouldnt have console.log statements in a PR. Make sure you run `npm run lint` and `npm run fix` because it should've yelled at you for this